README.md

```diff
@@ -59,7 +59,7 @@ To run the examples, you need a PiCaS account and access to a database (DB) on t
 <summary>Connect to the PiCaS server</summary>
 <br>
 
-To connect to the PiCaS server, fill `examples/picasconfig.py` with the information needed to log in to your PiCaS account and the database you want to use for storing the work tokens. Specifically, the information needed are:
+To connect to the PiCaS server, create a file `examples/picasconfig.py` with the information needed to log in to your PiCaS account and the database you want to use for storing the work tokens. A template can be found in `examples/picasconfig_example.py`. Specifically, the information needed are:
```

Comment:
Did you grep the examples that everywhere this name is changed?

Reply:
The examples still use the same config-file `examples/picasconfig.py`, but now this is not under version control anymore and the user has to create it themselves based on `examples/picasconfig_example.py`. This is to to avoid accidentally committing the config-file with credentials with `git add -u` for example.

Reply:
Makes sense, I only use `git add -p` nowadays.